spec/bundler/fetcher/compact_index_spec.rb

```diff
@@ -59,8 +59,7 @@ def remove_cached_md5_availability
 
         context "when FIPS-mode is active" do
           before do
-            allow(OpenSSL::Digest::MD5).to receive(:digest).
-              and_raise(OpenSSL::Digest::DigestError)
+            allow(OpenSSL).to receive(:fips_mode).and_return(true)
```

Comment:
this should return `false` but i'm also not sure if this should be changed, i don't think this will work on ruby 1.X versions

Reply:
I don't get why it should return `false`, the context is `context "when FIPS-mode is active" do`. Can you please explain?

Reply:
Because we're raising an exception with `and_raise(OpenSSL::Digest::DigestError)` in the `before` block.  We're also already testing the happy path on line 70, so the context seems to be inaccurate.

Reply:
Oh i got mixed up here, you're right.